draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -763,6 +761,31 @@ inactivity timer is implemented by the device to flush the context in case
 it receives nothing from the SCHC gateway over an extended period of time. The
 RECOMMENDED value is 12 hours for both Class B and Class C devices.
 
+## SCHC Fragment Format
+
+### All-0 SCHC fragment
+
+**Uplink fragmentation (Ack-On-Error)**:
+
+All-0 is distinguishable from a SCHC ACK REQ as [RFC8724] states *This condition
+is also met if the SCHC Fragment Header is a multiple of L2 Words*; this
+condition met: SCHC header is 2 bytes.
+
+**Downlink fragmentation (Ack-always)**:
+
+TBC
+
+### All-1 SCHC fragment
+
+All-1 is distinguishable from a SCHC Sender-Abort as [RFC8724] states *This
+condition is met if the RCS is present and is at least the size of an L2 Word*;
+this condition met: RCS is 4 bytes.
+
+### Delay after each message to respect local regulation
+
+This profile does not define a delay to be added after each SCHC message, local
+regulation compliance will be enforced by LoRaWAN stack.
```

Comment:
Maybe a slightly more precise version would be _regulation compliance is expected to be enforced by LoRaWAN stack._

Reply:
Thank you, fixed by bcd3993b5b26f05c1425aa8472e35b57a74965db